omod/src/main/java/org/openmrs/module/bedmanagement/extension/html/AdminList.java

```diff
@@ -28,7 +28,9 @@ public String getTitle() {
 	 */
 	public Map<String, String> getLinks() {
 		LinkedHashMap<String, String> map = new LinkedHashMap<String, String>();
-		String appBaseUrl = Context.getAdministrationService().getGlobalProperty(KEY_APP_BASE_URL);
+		String appBaseUrl = Context.getAdministrationService().getGlobalProperty(KEY_APP_BASE_URL) != null
+				? Context.getAdministrationService().getGlobalProperty(KEY_APP_BASE_URL)
+				: "/owa";
```

Comment:
This is a good idea but filling a default value for the GP is none if defined should rather go into the module's activator, [here](https://github.com/openmrs/openmrs-module-bedmanagement/blob/36cc08fa1533dc5e33dff8bdffee0b1fd458c52b/api/src/main/java/org/openmrs/module/bedmanagement/BedManagementActivator.java#L54).

Reply:
@sanishmaharjan did you see my comment above? You should set the GP in the module's activator once and for all. Not on the fly when you attempt to use it in the legacy UI.

Reply:
@mks-d Done. Set GP in the module's activator.